pytest_invenio/fixtures.py

```diff
@@ -554,3 +554,38 @@ def _get_screenshots_dir():
     if not os.path.exists(directory):
         os.makedirs(directory)
     return directory
+
+
+@pytest.yield_fixture(scope='function')
+def location(db):
+    """Sets up simple location"""
+    from invenio_files_rest.models import Location
+    uri = tempfile.mkdtemp()
+    location_obj = Location(name="pytest-location", uri=uri, default=True)
+
+    db.session.add(location_obj)
+    db.session.commit()
+
+    yield location_obj
+
+    shutil.rmtree(location_obj.uri)
+
+
+@pytest.fixture(scope="function")
+def bucket_from_dir(db, location):
```

Comment:
Since location includes the db fixture you don't need to add it separately. :)

Reply:
Yes, but I'm using db inside this fixture, and it's easier(and nicer) to pass it as a db fixture than import form other modules to this fixture.